enterprise/frontend/src/embedding-sdk/README.md

```diff
@@ -688,6 +688,27 @@ useEffect(() => {
 return <InteractiveQuestion key={counter} questionId={yourQuestionId} />
 ```
 
+### Customizing JWT authentication
+
+You can customize how the SDK fetches the refresh token by specifying the `fetchRefreshToken` function in the `config` prop:
+
+```jsx
+// This is the default implementation used in the SDK.
+// You can customize this function to fit your needs, such as adding headers or excluding cookies.
+// The function must return a JWT token object in the format of {id: string, exp: number}
+async function fetchRefreshToken(url) {
+  const response = await fetch(url, {
+    method: "GET",
+    credentials: "include",
+  });
+
+  return await response.json();
+}
```

Comment:
How should we handle errors?  throw?

For the case where the user hasn't logged in yet (and doesn't have a bearer token), can we return an empty or null payload?

Reply:
@sbrudz Thank you for the fast feedback! Yes, you can return `null` when there is no bearer tokens, and you can throw the error in this function. I'll update the docs and inline comments to clarify this 🙏🏻

Reply:
Great! Thank you!